src/elm/Update.elm

```diff
@@ -7,14 +7,15 @@ import Http
 import Maybe exposing (..)
```

Comment:
My gut feeling with common types like `Maybe`, `Task` and `Result` and functions like `map` would be to use them like `Maybe.map` rather than just `map`, it makes the intent much clearer. Didn't spot it on the last PR but this is one of the points https://github.com/njsfield/Nearest-Asteroids-Elm/blob/7271974f50801f6675824c79b2845de6ef78a701/src/elm/Update.elm#L155